announcing-python-sdk-ga.mdx

```diff
@@ -0,0 +1,69 @@
+
+## Speed improvements
+
+Under the hood, we've made adjustments to how connections are managed and reused. The refactoring has yielded significant performance improvements across the board.The most notable ones include accelerated operation speeds:
+
+| Operation | Speedup (on average)|
+| --- | --- |
+| Get a single record | 5.95x |
+| Insert a single record | 4.95x |
+| Insert 100 records with transactions | 2.22x |
+
+## How to migrate to 1.x?
+
+Migrating to a new major version can be tough due to the need to understand the changes that might break things and features that are no longer available. With that in mind, our goal was to minimize the impact while still implementing necessary changes for the greater good. You can check out the [full migration guide](/docs/python-sdk/migration-guide) in our docs.
+
+The most impactful user-facing change is the renaming of the API surface [[xata-py#93](https://github.com/xataio/xata-py/issues/93)]  which has been acknowledged as a breaking change. Additionally, some API endpoint calls were simplified to remove unnecessary code bloat, like creating a database.
+
+Previously in `0.x`, you needed to investigate the payload shape and be specific about the region and branch name.
+
+```python
+xata.databases().createDatabase("new_db",
+  {
+    "region": "us-east-1",
+    "branchName": "main",
+  }
+)
+```
+
+In version `1.0.*`, we've redesigned the API interface to incorporate payload options directly into the method signature. We've also optimized the SDK's internal values to make assumptions to reuse the SDK internal values, like the `region`. This results in a more streamlined API structure, and you can achieve functionality using:
+
+```python
+xata.databases().create("new_db")
+```
+
+## What happens to 0.x?
+
+We will continue providing support for the `0.x` version of the Xata Python SDK through maintenance releases for an additional year. During this time, new API enhancements and security fixes will be introduced; however, no backports of helpers or other improvements are planned. The `0.x` SDK version will be sunsetted by September 1st, 2024.
+
+## What’s next?
+
+Check out our [documentation](/docs/python-sdk/migration-guide) and let us know what you think. We’d love to hear from you! If you think something is missing or you found a bug, open a ticket in the [xata-py](https://github.com/xataio/xata-py) repository. All contributions are welcome. You can also follow us on [Twitter](https://twitter.com/xata) or join us in [Discord](https://xata.io/discord).
+
+If Go is your language of choice, our community has got you covered! The xata-go SDK, was initiated by [Kerdo](https://github.com/kerdokurs) and is hosted on [GitHub](https://github.com/kerdokurs/xata-go). Don't forget to check out [the story behind xata-go, a Go SDK for Xata](/blog/community-spotlight-xata-go-sdk).
```

Comment:
A quick intro sentence might be nice to provide context to this paragraph. You don't need to use this, but something like this eases folks into community contributions a bit more:

"Is there a language you wished we supported natively but don't today? Feel free to open up a [feature request](https://xata.canny.io/feature-requests) or check-in with our amazing community. The [xata-go SDK](/blog/community-spotlight-xata-go-sdk) was initiated by [Kerdo](https://github.com/kerdokurs) and is hosted on [GitHub](https://github.com/kerdokurs/xata-go). [mrkresnofatih](https://github.com/mrkresnofatih) also added the [terraform-provider-xata](https://github.com/mrkresnofatih/terraform-provider-xata) to provision Xata. If you're interested in contributing, feel free to reach out with any questions! Happy building 😄

Reply:
much better 🚀